build.zig

```diff
@@ -168,6 +168,7 @@ pub fn build(b: *std.Build) !void {
     const tracy = b.option([]const u8, "tracy", "Enable Tracy integration. Supply path to Tracy source");
     const tracy_callstack = b.option(bool, "tracy-callstack", "Include callstack information with Tracy data. Does nothing if -Dtracy is not provided") orelse (tracy != null);
     const tracy_allocation = b.option(bool, "tracy-allocation", "Include allocation information with Tracy data. Does nothing if -Dtracy is not provided") orelse (tracy != null);
+    const tracy_callstack_depth: c_int = b.option(c_int, "tracy-callstack-depth", "Declare callstack depth for Tracy data. Does nothing if -Dtracy_callstack is not provided") orelse 10;
```

Comment:
c_int is not appropriate here since it corresponds to the host target whereas I think you are trying to match the target C integer. Just pick a fixed width integer type please.

Reply:
Right I was trying to match the target C integer. 

Ok, changed to `u32` now.